feature/_version.py

```diff
@@ -2,4 +2,4 @@
 # Copyright FMR LLC <opensource@fidelity.com>
 # SPDX-License-Identifier: GNU GPLv3
 
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```

Comment:
It looks like we have /dist/ folder with a wheel? Should we drop that? I assume we moved to PyPI now (which reminds me that we shall update PyPI after merge?)

Reply:
Yes, can probably drop the wheel and will need to update PyPI after merge.

However, I think it probably still makes sense to include instructions on how to build from source in the README (which we do not have the moment). Should I add?

Reply:
Ah I see. Yeah compile instructions would help, if it's easy to add --but not a must for this PR

I recently did a pass on across all our GitHub libraries to make those sections coherent. But I think they point to the Documentation (which we don't have here).